flink-autoscaler-plugin-jdbc/src/test/java/org/apache/flink/autoscaler/jdbc/event/AbstractJdbcAutoscalerEventHandlerITCase.java

```diff
@@ -430,6 +446,70 @@ void testScalingEventWithParallelismChange() throws Exception {
                         });
     }
 
+    @MethodSource("getExpiredEventHandlersCaseMatrix")
+    @ParameterizedTest
+    void testCleanExpiredEvents(
+            int expiredRecordsNum, Duration eventHandlerTtl, int unexpiredRecordsNum)
```

Comment:
It's a little wired that `JdbcAutoScalerEventHandler#eventHandlerTtl` is always 0, but it could clean expired records.

Reply:
Thanks for comments.
IIUC, Maybe it's accepted, because in the current case, we just want to test the method `cleanExpiredEvents` of JdbcAutoScalerEventHandler instead of the whole logic of the `JdbcAutoScalerEventHandler`.
If needed, I'd like to  do some change for the stable test cases.